test/cli/build.js

```diff
@@ -433,6 +437,25 @@ test('`fusion build` with assets', async t => {
   t.end();
 });
 
+test('`fusion build` compresses assets for production', async t => {
+  const dir = path.resolve(__dirname, '../fixtures/compress-assets');
+  await cmd(`build --dir=${dir} --production`);
+
+  fs.readdir(
+    path.resolve(dir, '.fusion/dist/production/client/'),
+    (err, files) => {
+      if (err) throw err;
+      t.ok(files.filter(file => getFileExtension(file) === 'gz'), 'gzip works');
+      t.ok(
+        files.filter(file => getFileExtension(file) === 'br'),
+        'brotli works'
+      );
+      t.ok(files.filter(file => getFileExtension(file) === 'svg'), 'svg works');
```

Comment:
Is this testing SVGO? If so maybe check that the file contents don't have consecutive spaces

Reply:
Or I can just compare file sizes?

Reply:
yeah that works too

Reply:
Ah, that makes sense.